_data/namelink.yml

```diff
@@ -7,19 +7,18 @@
 'David Joyner': 'https://sites.google.com/site/wdjoyner/'
 'Derek Holt': 'http://www.maths.warwick.ac.uk/~dfh/'
 'Duane Broline': 'http://www.ux1.eiu.edu/~cfdmb/'
-"Eamonn O'Brien": 'http://www.math.auckland.ac.nz/~obrien'
```

Comment:
Huh, is this on purpose? I think Gap3/Datalib3/small.html still uses it

Reply:
I don't think that we wanted to delete him.

Reply:
Don't know why this happened.